### ISSUE TITLE
Improvements for tablet users

### ISSUE THREAD

**Issue by jeffras** [sdorra/angular-dashboard-framework#110]
Great project!! It would be great to improve the drag and drop functionality so that the dashboard is usable on mobile platforms. unfortunately arranging the dashboard is un-usable on these platforms

**Comment by sdorra** [comment]
You are right, but at the moment i'm not sure how to improve the navigation for tablets.

**Comment by ivan-saorin** [comment]
I think that D&D will never work great on a mobile device. Another metaphor
could be to select (tap on the widget) the widget to move and then select
the destination hotspot (tap another widget or a button after all the
widgets in a column). Thsi way everything would be feasible with to tap
actions.

2015-07-11 11:41 GMT+02:00 Sebastian Sdorra <notifications@github.com>:

**Comment by snoopyowns** [comment]
That would be a great solution Ivan. An user on a desktop would be able to use either method, but mobile devices that need to use click to move can. When it tested the framework on my phone, I was able to drag and drop just fine so I'm not sure if it is dependent on the device or browser. I was using a Samsung Galaxy S5 with Chrome.
